docs/zh/latest/how-to-build.md

```diff
@@ -21,146 +21,200 @@ title: 如何构建 Apache APISIX
+  ```
+
+4. 通过 `perl` 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集：
     * 追加当前目录到perl模块目录： `export PERL5LIB=.:$PERL5LIB`
     * 直接运行：`make test`
-    * 指定 nginx 二进制路径：`TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t`
+    * 指定 Nginx 二进制路径：`TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t`
     * 部分测试需要依赖外部服务和修改系统配置。如果想要完整地构建测试环境，可以参考 `ci/linux_openresty_common_runner.sh`。
 
-### 疑难排解测试
+### 问题排查
 
 **配置 Nginx 路径**
 
-如果遇到问题 `Error unknown directive "lua_package_path" in /API_ASPIX/apisix/t/servroot/conf/nginx.conf`
-确保将openresty设置为默认的nginx并按如下所示导出路径。
+出现`Error unknown directive "lua_package_path" in /API_ASPIX/apisix/t/servroot/conf/nginx.conf` 报错的解决方法如下：
+
+确保将 Openresty 设置为默认的 Nginx，并按如下所示导出路径：
 
 * export PATH=/usr/local/openresty/nginx/sbin:$PATH
   * Linux 默认安装路径：
     * export PATH=/usr/local/openresty/nginx/sbin:$PATH
-  * OSx 通过homebrew默认安装路径：
+  * MacOS X 通过 homebrew 默认安装路径：
     * export PATH=/usr/local/opt/openresty/nginx/sbin:$PATH
 
 **运行单个测试用例**
 
-- 使用以下命令运行指定的测试用例：
-  - prove -Itest-nginx/lib -r t/plugin/openid-connect.t
+使用以下命令运行指定的测试用例：
+
+```shell
+prove -Itest-nginx/lib -r t/plugin/openid-connect.t
+```
+
+## 步骤5：修改 Admin API token
 
-## 5. 更新 Admin API 的 token ，保护 Apache APISIX
+您需要修改 Admin API 的 token，以保护 Apache APISIX。
 
-修改 `conf/config.yaml` 中的 `apisix.admin_key` 并重启服务。例如下面例子：
+请修改 `conf/config.yaml` 中的 `apisix.admin_key` 并重启服务，如下所示：
 
 ```yaml
 apisix:
   # ... ...
   admin_key
     -
       name: "admin"
-      key: abcdefghabcdefgh
+      key: abcdefghabcdefgh # 将原有的token修改为abcdefghabcdefgh
       role: admin
 ```
 
 当我们需要访问 Admin API 时，就可以使用上面记录的 key 作为 token 了。
 
 ```shell
-$ curl http://127.0.0.1:9080/apisix/admin/routes?api_key=abcdefghabcdefgh -i
+curl http://127.0.0.1:9080/apisix/admin/routes?api_key=abcdefghabcdefgh -i
 HTTP/1.1 200 OK
 Date: Fri, 28 Feb 2020 07:48:04 GMT
 Content-Type: text/plain
 ... ...
 {"node":{...},"action":"get"}
 
-$ curl http://127.0.0.1:9080/apisix/admin/routes?api_key=abcdefghabcdefgh-invalid -i
+curl http://127.0.0.1:9080/apisix/admin/routes?api_key=abcdefghabcdefgh-invalid -i
 HTTP/1.1 401 Unauthorized
 Date: Fri, 28 Feb 2020 08:17:58 GMT
 Content-Type: text/html
 ... ...
 {"node":{...},"action":"get"}
 ```
 
-## 6. 为 APISIX 构建 OpenResty
+## 步骤6：为 APISIX 构建 OpenResty
 
-有些功能需要你引入额外的 Nginx 模块到 OpenResty 当中。
-如果你需要这些功能，你可以用[这个脚本](https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-openresty.sh)
-构建 OpenResty。
+有些功能需要引入额外的 Nginx 模块到 OpenResty 当中。如果您需要这些功能，你可以用[这个脚本](https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-openresty.sh)构建 OpenResty。
 
-## 7. 为 APISIX 添加 systemd 配置文件
+## 步骤7：为 APISIX 添加 systemd 配置文件
 
-如果通过 rpm 包安装 APISIX，配置文件已经自动安装到位，你可以直接运行
+如果您使用的操作系统是 CentOS 7，且在步骤 2 中通过 RPM 包安装 APISIX，配置文件已经自动安装到位，你可以直接运行以下命令：
 
-```
-$ systemctl start apisix
-$ systemctl stop apisix
-$ systemctl enable apisix
+```shell
+systemctl start apisix
+systemctl stop apisix
```

Comment:
We could remove this line if the need is to enable apisix (I made the mistake last time 😿

Reply:
ok :)